src/components/contextual/pages/vebal/LMVoting/GaugesFilters.vue

```diff
@@ -120,3 +120,9 @@ function updateNetwork(network: number) {
     </div>
   </BalPopover>
 </template>
+
+<style scoped>
+.filter-btn {
+  flex: 0 0 auto;
```

Comment:
Can this not be achieved with tailwind classes?

Reply:
yes, it can. I added class in `tailwind.config`